pages/agent/v3/hooks.md.erb

```diff
@@ -1,24 +1,46 @@
 # Buildkite Agent Hooks
 
-Every agent installer comes with a hooks directory which can be used to override and extend the built-in agent behavior. For example, you could create an agent `checkout` hook which speeds up a fresh `git clone` on a new build machine, an agent `environment` hook which exports secret API keys as environment variables, or a repository `pre-command` hook which sets up some repository-specific environment variables.
+Every agent installer comes with a hooks directory which can be used to override and extend the built-in agent behavior. For example, you could create an agent `checkout` hook which speeds up a fresh `git clone` on a new build machine, an agent `environment` hook which exports secret API keys as environment variables, or a repository `pre-command` hook which sets up repository-specific environment variables.
 
 {:toc}
 
 ## Types of hooks
 
 There are three types of hooks:
 
-1. [Agent hooks](#agent-hooks) - these exist on the agent’s machine, and are run for every job on any pipeline.
-1. [Repository hooks](#repository-hooks) - these exist in your pipeline repository’s `.buildkite/hooks` directory.
-1. [Plugin hooks](#plugin-hooks) - these are provided by any [plugins](/docs/plugins) you've defined in your `pipeline.yml`.
+* Agent hooks - these exist on the agent’s machine, and are run for every job on any pipeline.
+* Repository hooks - these exist in your pipeline repository’s `.buildkite/hooks` directory.
+* Plugin hooks - these are provided by any [plugins](/docs/plugins) you've defined in your `pipeline.yml`.
+
+### Agent hooks
+
+Each agent installer comes with a hooks directory containing a set of sample hooks. You can find the location of your agent hooks directory in your platform’s installation documentation.
+
+To get started with agent hooks copy the relevant example script and remove the `.sample` file extension.
+
+### Repository hooks
+
+Repository hooks allow you to execute repository-specific scripts. Repository hooks live alongside your repository’s source code under the directory `.buildkite/hooks`.
+
+To get started create a `.buildkite/hooks` directory, add your hook script, and make it executable.
+
+### Plugin hooks
+
+Plugin hooks allow any plugins you’ve defined to execute and override any of the default behavior.
```

Comment:
```suggestion
Plugin hooks allow any plugins you’ve defined as part of the pipeline step to override any of the default behavior.
```

Reply:
This is a good call. We don't currently lint for that because 

```
$ ack -ch execute*
819
```